src/elli.erl

```diff
@@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 
 remove_acceptor(State, Pid) ->
-    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
```

Comment:
Yes, I believe it will. And immediately be restarted by the supervisor,
which would create the table.

On Thu, Jun 11, 2015 at 12:54 PM, Andreas Hasselberg <
notifications@github.com> wrote:

> In src/elli.erl
> <https://github.com/knutin/elli/pull/90#discussion_r32209537>:
>
> > @@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
> >  %%%===================================================================
> >
> >  remove_acceptor(State, Pid) ->
> > -    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
>
> If someone upgrades to this version of elli on a running system it will
> crash on the first start/stop of an acceptor process, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/knutin/elli/pull/90/files#r32209537>.
>